src/extension.js

```diff
@@ -7,9 +7,8 @@ const fetch = require("node-fetch");
 const Url = require("url");
 
 function activate(context) {
-	const appDir = require.main
-		? path.dirname(require.main.filename)
-		: globalThis._VSCODE_FILE_ROOT;
+	const hasMain = "main" in require;
+	const appDir = hasMain ? path.dirname(require.main.filename) : globalThis._VSCODE_FILE_ROOT;

```

Comment:
We could also check whether `require.main` is not null/underfined.

Reply:
@be5invis  This merge doesn't make sense. Sorry.